src/main.rs

```diff
@@ -39,11 +40,18 @@ async fn main() {
     let wait_block_duration = 2;
 
     let provider: Provider<Http> = Provider::<Http>::try_from(eth_node.clone()).unwrap();
-    let radio_name: &str = "poi-crosschecker";
+    let radio_name: &str = "poi-crosschecker-radio";
```

Comment:
perhaps we should update 'poi-crosschecker-radio' references to just 'poi-radio'?

Reply:
also done ✅